lib/Controller/DocumentController.php

```diff
@@ -157,8 +157,21 @@ private function getDiscovery() {
 			$contact_admin = $this->l10n->t('Please contact the "%s" administrator.', [$wopiRemote]);
 
 			try {
-				$wopiClient = \OC::$server->getHTTPClientService()->newClient();
-				$discovery = $wopiClient->get($wopiDiscovery)->getBody();
+				// If we are sending query to built-in CODE server, we avoid using IClint::get() method
```

Comment:
```suggestion
				// If we are sending query to built-in CODE server, we avoid using IClient::get() method
```
Just a small typo in comment